lib/overcommit/configuration.rb

```diff
@@ -1,5 +1,6 @@
 require 'digest'
 require 'json'
+require 'English'
```

Comment:
Are you sure this is necessary? I'm able to access `$LOAD_PATH` in an IRB session without `require 'English'`.

Reply:
You probably have `English` loaded by IRB itself. I have been bitten by this previously, so now I require `English` whenever I use english global constants.